security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -17,6 +17,72 @@ To access the latest release, see xref:../../security/compliance_operator/co-man
+// For more information, see (link:https://issues.redhat.com/browse/CMP-3589[CMP-3589]).
+
+* In prior versions of Compliance Operator, an update of the network stack from Openshift-SDN to OVNKubernetes would trigger an alert from a complianceScan Instance for `sdn not found`. In this release, Compliance Operator has been updated to stop scanning for OpenShift-SDN when it has been replaced by OVNKubernetes. For more information, see (link:https://issues.redhat.com/browse/CMP-3591[CMP-3591]).
+
+* Previously, duplicate variables could be accidentally created in `TailoredProfile` and were not correctly detected by Compliance Operator. With this release, duplicate `setValues` in `TailoredProfile` are identified and generate a warning event from a compliance scan. 
+// For more information, see (link:https://issues.redhat.com/browse/CMP-3596[CMP-3596]).
+
+* In previous releases of Compliance Operator, the rule `audit-log-forwarding-uses-tls` would fail even if tls was enabled for `clusterloggerforwarder`. With this release, `audit-log-forwarding-uses-tls` shows `PASS` when tls is enabled for `clusterloggerforwarder`. For more information, see (link:https://issues.redhat.com/browse/CMP-3597[CMP-3597]).
+
+* In prior versions of Compliance Operator, for the rule `rhcos4-service-systemd-coredump-disabled`, no remediation was generated after scanning the cluster. In this release, remediation is provided for `rhcos4-service-systemd-coredump-disabled`. 
+// For more information, see (link:https://issues.redhat.com/browse/CMP-3599[CMP-3599]).
+
+* To periodically check for updates to containers, administrators use the configuration of `imagestream.spec.tags.importPolicy.scheduled`. This is useful when working with an external container image registry, to periodically re-import an image, for example, to get latest security updates. In prior versions of Compliance Operator, the rule to check the setting of `imagestream.spec.tags.importPolicy.scheduled` would return `FAIL` regardless. With this release, upon correct application of remediation, Compliance Operator no longer reports `FAIL`. 
+// For more information, see (link:https://issues.redhat.com/browse/CMP-3601[CMP-3601]).
+
+* In prior releases of Compliance Operator, when remediations were applied for updating TLS ciphers for the `kubeapi` server, the Compliance Operator did not have needed permissions to make the change. With this release, Compliance Operator is able to correctly remediate ciphers for the `kubeapi` server. For more information, see (link:https://issues.redhat.com/browse/CMP-3606[CMP-3606]).
+
+* In prior versions of Compliance Operator, the creation of Compliance Scans would cause problems with the checking of deprecated profiles. With this release, Compliance Operator checks for deprecated profiles are no longer failing. For more information, see (link:https://issues.redhat.com/browse/CMP-3613[CMP-3613]).
```

Comment:
Actually the key point is creating ComplianceScans directly with custom content images. The below description is a bit long. Feel free to update:

In prior versions of Compliance Operator, creating ComplianceScans directly with custom content images would fail during the profile deprecation check. The operator would error out with "Could not check whether the Profile used by ComplianceScan is deprecated", preventing advanced users from using custom content images. With this release, Compliance Operator gracefully handles cases where the ProfileBundle cannot be determined, logging an informational message instead of failing the scan.